response_operations_ui/views/reporting_units.py

```diff
@@ -189,7 +189,7 @@ def view_resend_verification(ru_ref, party_id):
     email = respondent['pendingEmailAddress'] if 'pendingEmailAddress' in respondent \
         else respondent['emailAddress']
 
-    return render_template('re-send-verification-email.html', ru_ref=ru_ref, email=email)
+    return render_template('re-send-verification-email.html', ru_ref=ru_ref, email=email, tab='reporting_units')
```

Comment:
My gut says this should be `active_tab` or something similar.  Just having a `tab` variable doesn't properly describe what it's doing at a glance

Reply:
And anothe gut feeling is that having tab here is wrong.  I think somewhere in either frontstage or response-operations-ui, in the template theres an 'if <thing> in url then add active class to tab' or something like that

Reply:
I'm not sure about basing it on the url, as it means two places to refactor if we change the URL for some reason, and can also be an issue with matching spuriously.

That said, I wouldn't necessarily pass it here either.